packages/core/admin/admin/src/content-manager/pages/EditView/EditViewPage.tsx

```diff
@@ -1,72 +1,53 @@
 import * as React from 'react';
 
-import { createSelector } from '@reduxjs/toolkit';
-import { Main, ContentLayout, Grid, GridItem, Flex, Box } from '@strapi/design-system';
 import {
-  CheckPermissions,
-  LinkButton,
+  Box,
+  Flex,
+  Grid,
+  GridItem,
+  Main,
+  Tab,
+  TabGroup,
+  TabPanel,
+  TabPanels,
+  Tabs,
+} from '@strapi/design-system';
+import {
   LoadingIndicatorPage,
   useNotification,
-  useTracking,
-  useRBAC,
-  Permission,
-  AllowedActions,
+  AnErrorOccurred,
+  CheckPagePermissions,
+  useQueryParams,
 } from '@strapi/helper-plugin';
-import { Layer, Pencil } from '@strapi/icons';
-import { Attribute } from '@strapi/types';
 import { useIntl } from 'react-intl';
-import { useNavigate, useLocation, useParams, Navigate } from 'react-router-dom';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
 
-import { InjectionZone } from '../../../components/InjectionZone';
-import { useTypedSelector } from '../../../core/store/hooks';
 import { useEnterprise } from '../../../hooks/useEnterprise';
 import { useOnce } from '../../../hooks/useOnce';
-import { ContentTypeFormWrapper } from '../../components/ContentTypeFormWrapper';
-import { DynamicZone } from '../../components/DynamicZone/Field';
-import { EditViewDataManagerProvider } from '../../components/EditViewDataManagerProvider/Provider';
-import { FieldComponent } from '../../components/FieldComponent';
-import { Inputs } from '../../components/Inputs';
-import { useLazyComponents } from '../../hooks/useLazyComponents';
-import {
-  generatePermissionsObject,
-  getFieldsActionMatchingPermissions,
-} from '../../utils/permissions';
+import { Form } from '../../components/Form';
+import { DocumentRBAC, useDocumentRBAC } from '../../features/DocumentRBAC';
+import { useDoc } from '../../hooks/useDocument';
+import { useDocumentLayout } from '../../hooks/useDocumentLayout';
+import { useSyncRbac } from '../../hooks/useSyncRbac';
 import { getTranslation } from '../../utils/translations';
 
-import { DeleteLink } from './components/DeleteLink';
-import { DraftAndPublishBadge } from './components/DraftAndPublishBadge';
 import { Header } from './components/Header';
 import { InformationBoxCE } from './components/InformationBoxCE';
-
-import type { RootState } from '../../../core/store/configure';
-import type { EditLayoutRow, FormattedLayouts } from '../../utils/layouts';
+import { InputRenderer } from './components/InputRenderer';
+import { transformDocument } from './utils/data';
+import { createDefaultForm } from './utils/forms';
 
 // TODO: this seems suspicious
-const CTB_PERMISSIONS = [{ action: 'plugin::content-type-builder.read', subject: null }];
+// const CTB_PERMISSIONS = [{ action: 'plugin::content-type-builder.read', subject: null }];
```

Comment:
should we remove both commented lines?

Reply:
Not for now, it'll be re-introduced in this feature